src/lib.rs

```diff
@@ -27,27 +27,27 @@ impl CuredString {
     }
 
     fn __richcmp__(&self, other: &str, op: CompareOp) -> PyResult<bool> {
-        match op {
-            CompareOp::Eq => Ok(self.0 == other),
-            CompareOp::Ne => Ok(self.0 != other),
-            _ => Ok(false),
-        }
+        Ok(match op {
+            CompareOp::Eq => self.0 == other,
+            CompareOp::Ne => self.0 != other,
+            _ => false,
+        })
     }
 
     fn __contains__(&self, other: &str) -> PyResult<bool> {
         Ok(self.contains(other))
     }
 
     fn __bool__(&self) -> PyResult<bool> {
-        Ok(self.0.len() > 0)
+        Ok(!self.0.is_empty())
     }
 
     fn __str__(&self) -> PyResult<&str> {
         Ok(&self.0)
     }
 
     fn __repr__(&self) -> PyResult<String> {
-        Ok(format!("CuredString(\"{}\")", &self.0))
```

Comment:
I prefer to keep this as is. calling str() on a cured string will just produce the string but calling repr() will provide information that its actually the CuredString class.

Reply:
The reason why i used this is because calling fmt::Debug ({:?}) on CuredString produces the exact same representation (`CuredString("contents")`) so i prefer just relying on the library here.

Reply:
Interesting, that works for me. I assumed it would have the rust struct { } syntax instead of parentheses.